packages/nest/src/app/oauth.module.ts

```diff
@@ -0,0 +1,74 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { SQLDialect, SQLInterface } from '@mobilejazz/harmony-core';
+import OAuth2Server = require('oauth2-server');
+
+import { DatabaseModule } from './database.module';
+import {
+    AuthControllerInteractor,
+    GetOAuthUserInteractor,
+    LoginOAuthUserInteractor,
+    OAuth2GuardInteractor,
+    OAuthSQLProvider,
+    ValidateScopeInteractor,
+} from '../oauth';
+
+// See: https://stackoverflow.com/a/52183279/379923
+interface Type<T> extends Function {
+    new (...args: unknown[]): T;
+}
+
+export interface OAuthModuleParams {
+    getUser: Type<GetOAuthUserInteractor>;
+    loginUser: Type<LoginOAuthUserInteractor>;
+    validateScope: Type<ValidateScopeInteractor>;
+}
+
+@Module({})
+export class OAuthModule {
+    static forRoot(params: OAuthModuleParams): DynamicModule {
+        return {
+            global: true,
+            module: OAuthModule,
+            imports: [
+                DatabaseModule,
+                // OauthUserInfoModule,
+                // forwardRef(() => UserModule),
```

Comment:
Remove commented code

Reply:
Removed in 03be4951877ce3fac6daa66674fb5d9e9ac6bbe9